server/views/tasks/_navigation.njk

```diff
@@ -1,19 +1,19 @@
 {%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
 
-{% macro navigation(activeTab) %}
+{% macro navigation(activeTab, apArea) %}
   {{
     mojSubNavigation({
       label: 'Sub navigation',
       items: [
         {
           text: 'Allocated',
-          href: paths.tasks.index({})+ "?allocatedFilter=allocated",
-          active: (activeTab === 'allocated' or activeTab|length === 0)
+          active: (activeTab === 'allocated' or activeTab|length === 0),
+          href: (paths.tasks.index({})+ "?allocatedFilter=allocated&area="+apArea if apArea else paths.tasks.index({})+ "?allocatedFilter=allocated")
```

Comment:
Minor thing but might be good to retain the spaces around the `+`s if this stays in the template